packages/modules/localstack/src/localstack-container.test.ts

```diff
@@ -68,5 +68,19 @@ describe("LocalStackContainer", () => {
     expect(response).toContain(`http://localstack:${LOCALSTACK_PORT}`);
     await container.stop();
     await awsCliInDockerNetwork.stop();
+    await network.stop();
+  });
+
+  it("should not override LOCALSTACK_HOST assignment", async () => {
+    const container = await new LocalstackContainer()
+      .withEnvironment({ LOCALSTACK_HOST: "myhost" })
+      .withNetworkAliases("myalias") // the last alias is assigned to LOCALSTACK_HOST by default
+      .start();
+
+    const { output, exitCode } = await container.exec(["printenv", "LOCALSTACK_HOST"]);
+    expect(exitCode).toBe(0);
+    expect(output).toContain("myhost");
```

Comment:
This is now just testing that the environment variables are passed through. Is there something about this bug that we can test?

Reply:
This pass-through is apparently not happening, which may be another bug.

Reply:
@cristianrgreco I changed the test to test the original bug, which is that LOCALHOST was being "overridden". 

But now I wonder if the overrides are working...

Reply:
Should `resolveHostname` be calling `withEnvironment` to make the assignment?